### DISCUSSION TITLE
输入hi，输出乱码的情况

### DISCUSSION THREAD

**Discussion by YuWLong666** [01-ai/Yi#543] [Q&A]
请问有朋友碰到过 LLM输入hi，输出乱码的情况吗，环境按照模型要求配置的。在L20服务器正常，H100上就乱码。查了一通，也没查到，网上也没搜到。

**Comment by Haijian06** [comment]
@YuWLong666 感谢提问，这个可能和推理框架 vllm 有一定的关系，您可以先尝试使用基本的 Transformers 进行推理，检查情况或者您可以去单独开一个 Issuse 我们会复现你的问题，再次感谢！
